src/libs/SidebarUtils.js

```diff
@@ -250,7 +249,7 @@ function getOptionData(reportID) {
         lastMessageTextFromReport = Str.htmlDecode(report ? report.lastMessageText : '');
     }
 
-    const lastActorDetails = personalDetailMap[report.lastActorEmail] || null;
+    const lastActorDetails = personalDetails[report.lastActorEmail] || null;
```

Comment:
This is the actual change that fixes the issue - the rest of the changes are just renaming variables in an attempt to make them more clear.

Reply:
Hmm...I'm a little fuzzy on why we are doing some of this.

 - Why do we bother building the `participantPersonalDetailList` if we already have all that information in the `personalDetails`?
 - If we build `participantPersonalDetailList` because the `getPersonalDetailsForLogins` method gets personal details that might not currently be in `personalDetails`, shouldn't we use `participantPersonalDetailList` for getting the `lastActorDetails` as well? We know `report.lastActorEmail` will be in `participantPersonalDetailList`, how do we know that `report.lastActorEmail` will always be in `personalDetails`?

Reply:
> Why do we bother building the participantPersonalDetailList if we already have all that information in the personalDetails?

Answered in the above comment!

> If we build participantPersonalDetailList because the getPersonalDetailsForLogins method gets personal details that might not currently be in personalDetails, . . .

Ah, that's not correct - take a look at the [implementation](https://github.com/Expensify/App/blob/90e49366d832ee8592077df482362b34245310d0/src/libs/OptionsListUtils.js#L100-L122) for `getPersonalDetailsForLogins` - all that method is doing is pulling the Personal Details we want from the `pesonalDetails` object in Onyx. So, all Personal Details in `participantPersonalDetailList ` will be in `personalDetails`.